kubetools_client/dev/backends/__init__.py

```diff
@@ -0,0 +1,22 @@
+import sys
+
+from kubetools_client.settings import get_settings
+
+from . import docker_compose
+
+
+DEV_BACKEND_PROVIDERS = {
+    'docker_compose': docker_compose,
+}
+
+
+# Now, using settings splice in the .backend module to match the provider
+settings = get_settings()
+
+try:
+    backend = DEV_BACKEND_PROVIDERS[settings.DEV_BACKEND]
```

Comment:
I'm guessing long run this will pull in from `kubetools.yml`?

Reply:
Unsure yet! So thinking it's more "machine specific" so would be part of `~/.kubetools/kubetools.conf` rather than project specific. 

Ideally this would mean two engineers could work on the same project - one using docker compose as is today and another using k3s; all without any specific changes depending on which one.

Reply:
Ah I see! Is there something driving that or just as a nice to have kind of feature?

Reply:
Very much nice-to-have! Saw a thing about local dev with K3S and seems sensible to be flexible w.r.t this!